app/models/scrum_backlog.rb

```diff
@@ -1,76 +1,55 @@
 class ScrumBacklog < ApplicationRecord
```

Comment:
This model was really renamed and then recreated to reflect a different concept all together.

I'll need to update the README to explain the basic concepts/models. I was originally conceptualizing the whole board as backlog. But then there is the backlog column. And the current sprint column is also referred to as the sprint backlog. So to limit confusion, I renamed the `ScrumBacklog` model to `ScrumBoard`.